src/main/scala/org/allenai/plugins/DeployPlugin.scala

```diff
@@ -482,9 +470,8 @@ object DeployPlugin extends AutoPlugin {
     * that should be shared between them, when deploying a project.
     * @param hostConfigs configuration objects specifying how the project should be deployed to
     *                    many hosts
-    * @param projectVersion the version of the project being deployed
     */
-  case class DeployConfig(hostConfigs: Seq[HostConfig], projectVersion: Option[String])
+  case class DeployConfig(hostConfigs: Seq[HostConfig])
```

Comment:
This case class is superfluous within the scope of this PR, but I left it in because I'll add additional fields to it as part of subsequent replica changes.